### PR TITLE
Design and build 2

### PR DESCRIPTION
**Pull request by jfidiles**

### CODE REVIEW

File: _sass/main/containers/_stepper-container.scss [hunk 1]
```diff
@@ -62,7 +62,7 @@ $icon-stepper-size: 30px;
 }
 
 .StepperSeparator {
-  height: 50px;
+  height: 50px !important;
```
Comment (artisologic) [inline _sass/main/containers/_stepper-container.scss]: Why did you need to override specificity here?

File: assets/images/design-and-build/cloud-dark.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg width="46" height="30" viewBox="0 0 46 30" xmlns="http://www.w3.org/2000/svg">
```
Comment (artisologic) [inline assets/images/design-and-build/cloud-dark.svg]: Just a note, but at some point we should refactor all these SVG's to take advantage of `currentColor` rather than have different versions of the files with different hardcoded colors.

Comment (jfidiles) [inline assets/images/design-and-build/cloud-dark.svg]: Noted

File: design-and-build.html [hunk 3]
```diff
@@ -3,46 +3,166 @@
...
-                </p>
-
-                <p>
-                    We design intuitive experiences and beautiful, highly efficient user
-                    interfaces which simplify this complexity.
-                </p>
-            </section>
-        </div>
+<div class="DarkContainer">
+    <div class="ContentContainer SubjectContainer">
```
Comment (peduarte) [inline design-and-build.html]: Too many `Containers` at different levels. This creates a very confusing layout API. Eventually it'll be come hard to know which element is actually responsible for a Container behaviour. I think we can make this better. You can show me what you're currently doing and we can refactor it. Can be a separate PR

File: design-and-build.html [hunk 4]
```diff
@@ -3,46 +3,166 @@
...
-    <div class="ImageContainer">
-        <img src="assets/images/design-and-build/background1.png" alt="" />
+<div class="LightContainer">
+    <div class="ContentContainer">
+        <div class="StepperContainer-title">
+            <h3>
+                Sed ut perspiciatis
+                <strong>unde omnis iste</strong>
+                natus error sit voluptatem accusantium doloremque laudantium.
+            </h3>
```
Comment (peduarte) [inline design-and-build.html]: Too much text for a Heading here. Maybe should be `p`?
